recipes/qt/6.x.x/conanfile.py

```diff
@@ -238,7 +238,7 @@ def requirements(self):
         if self.options.get_safe("with_vulkan"):
             self.requires("vulkan-loader/1.2.172")
             if tools.is_apple_os(self.settings.os):
-                self.requires("moltenvk/1.1.1")
+                self.requires("moltenvk/1.1.4")
                 self.requires("vulkan-headers/1.2.172", override=True)
```

Comment:
MoltenVK 1.1.4 is for Vulkan 1.2.182

Reply:
ok, I will then bump the vulkan-loader version